### PR TITLE
Agabrie

### PR DESCRIPTION
**Pull request by agabrie**
Created A Generic Enemy class, Will change direction when colliding with another object OR when 8 seconds have passed since the last time it autoswitched

### CODE REVIEW

**Review by TalPat — COMMENTED** [review]
Maybe merge your branch with master first then resolve conflicts, then redo the pull request?

**Comment by WSeegers** [comment]
The pull request will be updated if you fix your branch. Merge master

**Comment by charlchi** [comment]
@agabrie While in your `agabrie` branch, do a `git pull https://github.com/TalPat/Bomberman.git master`. All the conflicts should be resolved automatically and we can merge this.

**Review by charlchi — APPROVED** [review]

File: models/src/Enemy.cpp [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+	}
+	return false;
+}
+
+void Enemy::changeMoveState(){
+	int max = 3;
+	int min = 0;
+	int num;
+	num = (rand() % (max + 1 - min)) + min;
+	// std::cout << num <<"\n";
```
Comment (WSeegers) [inline models/src/Enemy.cpp]: Remove all these commented debug statements.

**Review by WSeegers — CHANGES_REQUESTED** [review]
Remove the .DS_Store(s) and add this to the .gitignore.
Clean up the the debug comments please.

File: models/src/Enemy.cpp (hunk 1 above)
Comment (WSeegers) [reply models/src/Enemy.cpp]: ```models/.DS_Store``` still exists

**Review by WSeegers — CHANGES_REQUESTED** [review]
Remove ```models/.DS_store```

**Review by WSeegers — COMMENTED** [review]
Looks good

**Review by WSeegers — APPROVED** [review]

**Pull request merged**